lambda_function/aws/resources/elastic_cloud_compute_handler.py

```diff
@@ -0,0 +1,101 @@
+import json
+import logging
+from collections import OrderedDict
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import boto3
+import consts
+import yaml
+from aws.resources.base_handler import BaseHandler
+from port.entities import create_entities_json, handle_entities
+
+logger = logging.getLogger(__name__)
+
+
+class EC2Handler(BaseHandler):
```

Comment:
I think the name should be EC2InstanceHandler , for compatibility with other ec2 resources

Reply:
Sure, I'll do that